src/controller/index.ts

```diff
@@ -352,7 +350,8 @@ export default class SimulariumController {
     public changeFile(
         connectionParams: SimulatorConnectionParams,
         // TODO: push newFileName into connectionParams
-        newFileName: string
+        newFileName: string,
+        keepRemoteConnection: boolean = false
```

Comment:
Linting had a comment here to remove boolean
```suggestion
        keepRemoteConnection = false
```

Reply:
I'm learning new things about typescript every day! it makes sense that you don't have to explicitly define the type in this situation

- [ ] lint fix